curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/returning-boolean-values-from-functions.english.md

```diff
@@ -59,12 +59,13 @@ tests:
 
 ```js
 function isLess(a, b) {
-  // Fix this code
+  // Only change code below this line
   if (a < b) {
     return true;
   } else {
     return false;
   }
+  // Only change code above this line
 }
 
 // Change these values to test
```

Comment:
```suggestion
```
Might as well remove this for consistency.

Reply:
I actually do remove these in one of the PRs.  Because of the way I worked on these and split them up into separate PRs (to make it easier to review), some things get removed/consolidated on different PRs (even when the changes are in the same file).  Once all 4 of these have been merged, then I will go back through (I have a script that finds all the comments) and validate everything was captured.

Reply:
Okay, what I'll do is look for cases where something has been removed that should not have been.  I'll not worry about anything that's still there.